Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -268,6 +278,15 @@ extension ActivityStreamPanel {
         }
     }
 
+    private func hideURLFromTopSites(siteURL: NSURL) {
+        guard let host = siteURL.normalizedHost() else {
+            return
+        }
+        profile.history.removeHostFromTopSites(host).uponQueue(dispatch_get_main_queue()) { result in
```

Comment:
Nit: Don't need to do an `uponQueue` if there isn't anything being done in the upon block. Just `profile.history.removeHostFromTopSites(host)` works.